exporters/cht-user-management/scrape.yml

```diff
@@ -0,0 +1,7 @@
+scrape_configs:
+  - job_name: cht-user-management-metrics
+    metrics_path: /metrics
```

Comment:
The [default path](https://github.com/prometheus/prometheus/blob/0bfbd462ace2e24342a834a1aa89a6feecce1f99/documentation/examples/prometheus.yml#L25) is `/metrics` so we can drop this for simplicity:

```suggestion
```

Reply:
per other comment, still seeing `metrics_path: /metrics` in there - so assume local changes weren't pushed